subsys/pm/device_system_managed.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SUBSYS_PM_DEVICE_SYSTEM_MANAGED_H_
+#define ZEPHYR_SUBSYS_PM_DEVICE_SYSTEM_MANAGED_H_
+
+#ifdef CONFIG_PM_DEVICE_SYSTEM_MANAGED
+
+bool pm_suspend_devices(void);
+void pm_resume_devices(void);
+
+#else
+
+bool pm_resume_devices(void) { return true; }
+void pm_suspend_devices(void) {}
```

Comment:
@ceolin, is this a bug? Why are the function definitions different in the #if and #else cases?